### ISSUE TITLE
Easy Anti-Cheat dislikes devreorder's dinput8.dll (in both the game directory and system-wide)

### ISSUE THREAD

**Issue by doctrewho** [briankendall/devreorder#33]
Probably nothing that can be done about that since it's not a "signed" dll but it's preventing me from playing Star Wars: Squadrons as I have to reorder my devices to get the correct ones to show up in Squadrons as it only picks up the first 2 input devices.

**Comment by briankendall** [comment]
Yeah I had the same issue with Vermintide II. The only option is to temporarily put the original dinput8.dll back in system32 while playing the game.

I'm thinking I might make a little utility for this, perhaps some kind of launcher, to at least automate this process.

**Comment by Cavermendes** [comment]
Did you ever get that utility setup? I can't play Fall guys and Rust because of stupid Easy Anti-Cheat.

**Comment by Z-K-O** [comment]
Star Citizen has the same issue since it started using EAC. Is there any way to trick EAC into thinking the file is legitimate?

**Comment by briankendall** [comment]
I don't know how the anti-cheat systems work, but my guess is that they do a checksum of various system DLLs (like dinput8.dll) to make sure they're the legit copies. If that's the case, then no, there's no way to trick them.

**Comment by ceo16** [comment]
![16729338121412411078001414147471](https://user-images.githubusercontent.com/990721/210822779-173cdd8b-0307-4d74-bdf7-12d529064438.jpg)
I had managed to solve all my problems with devreorder.... But I have the same problem as you with Fortnite
